doc/jsk_topic_tools/scripts/or_node.rst

```diff
@@ -0,0 +1,38 @@
+or_node.py
+==========
+
+
+What is this?
+-------------
+
+A node that takes bool values and returns the result of the ``or`` operation.
+
+For example, this node can be used to merge the results of a node that outputs whether the robot is speaking in English or Japanese.
+
+
+Subscribing Topics
+------------------
+
+* ``~input01`` (``std_msgs/Bool``)
+
+  input bool value.
+
+* ``~input02`` (``std_msgs/Bool``)
+
+  input bool value.
+
+
+Publishing Topics
+-----------------
+
+* ``~output`` (``std_msgs/Bool``)
+
+  The result of the ``or`` operation.
+
+
+Parameters
+----------
+
+* ``~rate`` (Int, Default: ``100``)
+
+  Publishing rate [Hz].
```

Comment:
Could you add usege of the node with topic remapping?

```
rosrun jsk_topic_tools xxx input01:=yy
```

Or, I think creating launch is also useful. (both is OK)

```
roslaunch jsk_topic_tools xxx.launch input01:=yyy 
```

Reply:
Added.